frontend/src/components/Staffing/DetailedBookingRows.tsx

```diff
@@ -87,29 +87,30 @@ interface updateBookingHoursProps {
   hours: number;
   bookingType: BookingType;
   organisationUrl: string;
-  consultantId: string;
-  bookingId: string;
+  consultantId: number;
+  bookingId: number;
   startWeek: number;
   endWeek?: number;
 }
 
 async function setDetailedBookingHours(props: updateBookingHoursProps) {
   const url = `/${props.organisationUrl}/bemanning/api/updateHours`;
-  const body: updateBookingHoursBody = {
+  const body: StaffingWriteModel = {
     hours: props.hours,
-    bookingType: props.bookingType,
+    type: props.bookingType,
     consultantId: props.consultantId,
-    bookingId: props.bookingId,
+    engagementId: props.bookingId,
     startWeek: props.startWeek,
-    endWeek: props.endWeek,
+    startYear: 2023, // TODO: Fix mismatch between front- and backend types
+    //endWeek: props.endWeek,
```

Comment:
This will maybe leave the drag function useless? as it needs an end week

Reply:
The reason for a different type here and in the route handler was to enable the use of a singular route handler for two different api end points with different types.